### DISCUSSION TITLE
Alpine seems to send a request for text/html.

### DISCUSSION THREAD

**Discussion by SickAssApp** [alpinejs/alpine#4217] [1. Help] [answered]
Hi I ran into a strange issue which I'm not sure if this is caused by Alpine or Laravel.
![issue01](https://github.com/alpinejs/alpine/assets/6093611/aab32837-c827-4175-8b77-32c52acc5d7e)

From picture above we can see red line is a request for page itself and the blue one is a request send again who's initiator is from Alpine.
When I clicked and go to the line in initiator I see following script
`function setIfChanged(el, attrName, value) {
  if (el.getAttribute(attrName) != value) {
    el.setAttribute(attrName, value);
  }
}`

What cause this to happen?

**Comment by ekwoka** [comment, marked as answer]
This will be from something in your code, just due to how Alpine works, it will appear to be initiated by Alpine.

Here maybe you have something that is like `:src="image"` or something else like that. Alpine set the attribute, and then the attribute requested the path.

**Reply by SickAssApp** [reply]
Hi thanks a lot it is caused by :src.
